samples/batch/python/README.md

```diff
@@ -8,25 +8,13 @@ Follow these steps for the installation:
 
 1. Go to https://editor.swagger.io.
 1. Click **File**, then click **Import URL**.
-1. Enter the Swagger URL including the region for your Speech Services subscription: `https://<your-region>.cris.ai/docs/v2.0/swagger`.
+1. Enter the Swagger URL for the Speech Services API: `https://westus.dev.cognitive.microsoft.com/docs/services/speech-to-text-api-v3-0/export?DocumentFormat=Swagger&ApiName=Speech%20to%20Text%20API%20v3.0`.
```

Comment:
is the "dev" in "westus.dev.xxxx" expected?

Reply:
Yes, that's the one we link to also from https://docs.microsoft.com/en-gb/azure/cognitive-services/speech-service/batch-transcription.